commons/h5bench_util.h

```diff
@@ -7,6 +7,7 @@
 
 #ifndef COMMONS_H5BENCH_UTIL_H_
 #define COMMONS_H5BENCH_UTIL_H_
+#include <hdf5.h>
```

Comment:
why do we need this header here?

Reply:
In the `h5bench_utils.h` file, hid_t is used in some signatures. So if you compile it separately with some extension (which I often did, because compiling the whole h5bench during development takes a lot of time), it causes errors: 
```
../commons/h5bench_util.h:171:5: error: unknown type name ‘hid_t’
  171 |     hid_t              es_meta_create;
      |     ^~~~~
../commons/h5bench_util.h:172:5: error: unknown type name ‘hid_t’
  172 |     hid_t              es_meta_close;
      |     ^~~~~
../commons/h5bench_util.h:173:5: error: unknown type name ‘hid_t’
  173 |     hid_t              es_data;
      |     ^~~~~
../commons/h5bench_util.h:174:5: error: unknown type name ‘hid_t’
  174 |     hid_t              grp_id;
      |     ^~~~~
../commons/h5bench_util.h:175:5: error: unknown type name ‘hid_t’
  175 |     hid_t              dset_ids[8];
      |     ^~~~~
../commons/h5bench_util.h:192:18: error: unknown type name ‘hid_t’; did you mean ‘id_t’?
  192 | void async_sleep(hid_t es_id, duration sleep_time);
      |                  ^~~~~
      |                  id_t
../commons/h5bench_util.h:207:1: error: unknown type name ‘hid_t’; did you mean ‘id_t’?
  207 | hid_t es_id_set(async_mode mode);
      | ^~~~~
      | id_t
../commons/h5bench_util.h:208:19: error: unknown type name ‘hid_t’; did you mean ‘id_t’?
  208 | void  es_id_close(hid_t es_id, async_mode mode);
      |                   ^~~~~
      |                   id_t

```
I can remove the include if you think it's better.

Reply:
Got it, let's keep it. Not sure why this was not caught in our CI.